content/en/blog/2025/ottl-contexts-just-got-easier.md

```diff
@@ -0,0 +1,153 @@
+---
+title: OTTL contexts just got easier with context inference
+linkTitle: OTTL contexts just got easier
+date: 2025-02-17
```

Comment:
We'd like to time this with releasing after the next collector release. Realistically it should definitely be out by Feb 20th.